learn/slides/optaplanner-presentation/index.html

```diff
@@ -993,45 +997,78 @@ <h3>DEMO</h3>
+            <img src="QuickStart/SchoolTimetabling/schoolTimetablingArchitecture_1.png" class="fullImage fragment">
         </section>
         <section>
-            <figure>
-                <img src="QuickStart/SchoolTimetabling/schoolTimetablingScreenshot.png"/>
-            </figure>
+            <h3>Project setup</h3>
+            <p>code.quarkus.io</p>
+            <p>mvn compile quarkus:dev</p>
         </section>
         <section>
-            <img src="PlannerIntroduction/useCaseOverview_0.png" class="fullImage">
-            <img src="PlannerIntroduction/useCaseOverview_1.png" class="fullImage fragment">
-            <img src="PlannerIntroduction/useCaseOverview_2.png" class="fullImage fragment">
-            <img src="PlannerIntroduction/useCaseOverview_3.png" class="fullImage fragment">
-            <img src="PlannerIntroduction/useCaseOverview_4.png" class="fullImage fragment">
-            <img src="PlannerIntroduction/useCaseOverview_5.png" class="fullImage fragment">
+            <h3>Domain</h3>
         </section>
         <section>
-            <img src="QuickStart/SchoolTimetabling/schoolTimetablingInputOutput_0.png" class="fullImage">
-            <img src="QuickStart/SchoolTimetabling/schoolTimetablingInputOutput_1.png" class="fullImage fragment">
-            <img src="QuickStart/SchoolTimetabling/schoolTimetablingInputOutput_2.png" class="fullImage fragment">
+            <img src="QuickStart/SchoolTimetabling/schoolTimetablingClassDiagram_0.png" class="fullImage">
+            <img src="QuickStart/SchoolTimetabling/schoolTimetablingClassDiagram_1.png" class="fullImage fragment">
+        </section>
+        <section>
+            <h3>UI</h3>
+        </section>
+        <section>
+            <img src="QuickStart/SchoolTimetabling/schoolTimetablingArchitecture_1.png" class="fullImage">
+            <img src="QuickStart/SchoolTimetabling/schoolTimetablingArchitecture_2.png" class="fullImage fragment">
+        </section>
+        <section>
+            <h3>UI in JavaScript</h3>
+            <p>OK...</p>
+        </section>
+        <section>
+            <img src="extra/Presentation/javaScriptLiveCoding_0.png" class="fullImage">
+            <img src="extra/Presentation/javaScriptLiveCoding_1.png" class="fullImage fragment">
+            <img src="extra/Presentation/javaScriptLiveCoding_2.png" class="fullImage fragment">
+            <img src="extra/Presentation/javaScriptLiveCoding_3.png" class="fullImage fragment">
+            <img src="extra/Presentation/javaScriptLiveCoding_4.png" class="fullImage fragment">
+            <img src="extra/Presentation/javaScriptLiveCoding_5.png" class="fullImage fragment">
+            <img src="extra/Presentation/javaScriptLiveCoding_6.png" class="fullImage fragment">
+        </section>
+        <section>
+            <h3>Coding a UI in JavaScript?</h3>
+            <p>Not in public</p>
+        </section>
```

Comment:
It's a joke, ok. Is it necessary? I don't know. No clear opinion about including this part - despite I share the opinion about javascript.

Reply:
This is 30 minutes into the presentation. According to the book "presentations patterns", you need a "brain break" (=joke) every 20 mins or so. It's the only one so far :)
But it could be an "alienating artifact" (same book) to JS fanboys, which is an anti-pattern.

Reply:
I can live with pissing off JS fanboys. ;-)

(just kidding)

Reply:
Yep, I wanted to point this out - decision on including it is up to you.

Reply:
@yurloc @triceo wdyt? is that anti-JS joke unprofessional?

Reply:
I'll merge it for now (people are waiting for the slides), but I am happy to take it out in a separate PR.